jython/DCC-EX/RosterExportToDCC-EX.py

```diff
@@ -0,0 +1,54 @@
+# RosterExportToDCC-EX.py, v1.1.0, 04/14/2023
+# Export all JMRI Roster Entries as ROSTER() macro calls
+#  Syntax: ROSTER(999,"Loco Name","F0/F1/*F2/F3/F4/F5/F6/F7/F8")
+#  Run the script. It will open the Script Output Window with the calls listed
+#  Copy the lines into your myAutomation.h and compile+upload DCC-EX to your Arduino.
+#  For convenience, you can "add script to button..." in Preferences, Startup
+# NOTE: EX-CommandStation cannot (yet) handle multiple roster entries with the same address
+# Author: mstevetodd, copyright 2023
+# Part of the JMRI distribution
+
+import jmri.jmrit.roster
+
+# check for Script Output window and open and show if not found
+f = jmri.util.JmriJFrame.getFrame("Script Output")
+if (f is None) :
+  a = jmri.jmrit.jython.JythonWindow("")
+  a.actionPerformed(None)
+  f = a.getFrame();
+f.setExtendedState(java.awt.Frame.NORMAL) #restore if minimized
```

Comment:
You might want a `f.setVisible(True)` here in case the window exists but has been hidden.

Reply:
I had that, but it didn't seem to make any difference. How would I make a window not visible for testing?

Reply:
Try in the script input window:
```
f = jmri.util.JmriJFrame.getFrame("Script Output")
f.setVisible(False)
```